app/views/application_templates/show.haml

```diff
@@ -50,8 +50,12 @@
               .col-md-6
                 .label= u.label field
               .col-md-6
-                = u.text_field field, size: 50, required: true,
-                  value: @current_user.try(field)
+                - if field == :email
```

Comment:
Wouldn't it be simpler to just not put it in the `.each`? I know someone was trying to save some repetition but...